reana-ui/src/components/Announcement.js

```diff
@@ -0,0 +1,47 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2020 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import React, { useState } from "react";
+import { Icon } from "semantic-ui-react";
+import { useSelector } from "react-redux";
+
+import { getAnnouncement } from "../selectors";
+
+import styles from "./Announcement.module.scss";
+
+export default function Announcement() {
+  const hiddenAnnouncement = window.localStorage.getItem("hideAnnouncement");
+  const announcement = useSelector(getAnnouncement);
+  const [visible, setVisible] = useState(
```

Comment:
Shouldn't it be `hidden` and `setHidden` instead of `visible` and `setVisible`?

Reply:
Indeed, I don't know what I was thinking... 🤦